scripts/beck/st/error_injection.py

```diff
@@ -295,62 +335,77 @@ def induce512kbError():
 				if j > 524288:
 					break
 				else:
-					wbfr.seek(offset)
-					wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
-					#ReadMetaData(offset)
+					EditMetadata(offset)
+					#wbfr.seek(offset)
+					#wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
+					count = count + 1
+	return count
 
 		
 def verifyLengthOfRecord(recordDict):
-	print("***********Record list will be print here************")
+	count = 0
+	read_metadata_file()
+	logger.info("***********Record list will be logger.info here************")
 	for record,items in recordDict.items():
-		print(" {} :  {}".format(record, len(items)))
-
-	
-if args.err512k:
-	induce512kbError()
-	
-if args.hugeCorruption:
-	induceHugeError()
+		logger.info(" {} :  {}".format(record, len(items)))
+		count = count + 1
+	return count
 	
-
-with open(filename, "rb") as metadata:
-	if os.path.getsize(filename):
+def read_metadata_file():
+	with open(filename, "rb") as metadata:
+		#if os.path.getsize(filename):
 		i=0
 		while 1:
-			flag=0
 			byte=metadata.read(8)
 			i=i+8
 			if not byte:
 				break
 			byte=binascii.hexlify(byte[::-1])
 			if byte == header:
-				byte=binascii.hexlify((metadata.read(8))[::-1])					#Read the Type Size Version
+				byte=binascii.hexlify((metadata.read(8))[::-1])			#Read the Type Size Version
 				i=i+8
```

Comment:
This and everywhere else:
i = i + 8
Please follow the coding style.
one space between variable, =, +